utils/dataset_manifest/core.py

```diff
@@ -18,8 +18,8 @@
 import av
 from PIL import Image
 
+from ._types import NamedBytesIO
```

Comment:
I think it's better to change how the `types.py` file is visible to python instead - it should not be visible as `types`. Consider changing the `PYTHONPATH` instead, similarly to what I tried in the PR linked.

Reply:
Modifying path inside the script - isn't. However, this issue is only relevant to direct `create.py` calls (as it's the only callable file here) and only because this component isn't organized as a regular module. So by prepending the parent directory into the pythonpath we just pretend it's a normal module called `dataset_manifest`, as it's supposed to be.

Reply:
In case of a name conflict with a built-in name usually an additional `_` is added as a suffix, not a prefix BTW. I agree with Oleg, it's better to change a file name than a way how this file is visible to interpreter. Because this may cause side effects

Reply:
Which side effects?